From 121136ca2b651135e6599ad0e33951ac2d7737a0 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 10 Feb 2020 16:02:06 +0200
Subject: [PATCH] [SW-0] goya2: pll: remove hl_pll_is_configurable()

Since the tests were run and proved there is no need to configure the
CPU PLL prior to waking the CPU (since the clock switch has been fixed
in H/W), there is no more need for the Goya2 private implementation of
hl_pll_is_configurable() function.
We still do not add the CPU PLL configuration to preboot, just remove
the protection.

Change-Id: Ie99a4ad7b5f8b2ec2edf38877aed09576e0dc2f5
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya2/asic.c b/goya2/asic.c
index 94b116f..00b27ea 100644
--- a/goya2/asic.c
+++ b/goya2/asic.c
@@ -156,18 +156,6 @@
 	},
 };
 
-bool hl_pll_is_configurable(struct hl_pll_cfg *cfg)
-{
-	switch (cfg->base) {
-	case mmPSOC_CPU_PLL_BASE:
-		return false;
-	default:
-		break;
-	}
-
-	return true;
-}
-
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
 {
 	int pll_num = ARRAY_SIZE(goya2_pll_cfg);
